src/main/scala/ru/wow42/tasks/futures/FutureTask.scala

```diff
@@ -1,13 +1,15 @@
+package ru.wow42.tasks.futures
+
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
-case class Test[A]() {
+
+case class FutureTask[A]() {
   def traverse(list: List[Future[A]]): Future[(List[Throwable], List[A])] = {
```

Comment:
Let's agree that the DoD (definition of done) of the any task is a set of green unit tests. I mean that task is done only when we have enough tests to check correctness of the implementation.